seq_logger.js

```diff
@@ -148,12 +139,19 @@ class SeqLogger {
     }
 
     _toWireFormat(event) {
+        const level = typeof event.level === 'string' ? event.level : undefined;
+        const timestamp = event.timestamp instanceof Date ? event.timestamp : new Date();
+        const messageTemplate = typeof event.messageTemplate === 'string' ? event.messageTemplate :
+            event.messageTemplate ? event.messageTemplate.toString() : "(No message provided)";
```

Comment:
Should this be `event.messageTemplate != undefined` (or `event.messageTemplate !== undefined && event.messageTemplate !== null`)? Otherwise we'll accept `"1"` as a message template, but will send `"(No message provided)"` instead of `"0"`.